api/Navigator.json

```diff
@@ -93,7 +93,7 @@
             },
             "samsunginternet_android": {
               "version_added": true,
-              "notes": "Currently supported only by Google Daydream."
+              "notes": "Only supported only by Google Daydream."
```

Comment:
I guess here the second "only" needs to be removed or you keep the original wording?